data/json/construction.json

```diff
@@ -1019,6 +1019,21 @@
     "pre_special": "check_support",
     "post_terrain": "t_floor"
   },
+  {
+    "type": "construction",
+    "description": "Build Thatched Roof",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 3 ], [ "survival", 3 ] ],
+    "time": 120,
```

Comment:
https://www.higos.co.uk/blog/how-long-does-it-take-to-thatch-a-roof

Thatching is extremely labour intensive, thatching a small cottage takes about a month with two or three people working full time. We can bend that for playability, but two hours seems insanely fast.